test/archethic/transaction_chain/transaction_input_test.exs

```diff
@@ -0,0 +1,28 @@
+defmodule Archethic.TransactionChain.TransactionInputTest do
```

Comment:
I think there is already a TransactionInputTest, located at `test/archethic/transaction_chain/transaction/input_test.exs`